pkg/util/containers/metrics/cgroup_metrics.go

```diff
@@ -131,6 +131,18 @@ func (c ContainerCgroup) MemLimit() (uint64, error) {
 	return v, nil
 }
 
+// FailedMemoryCount returns the number of times this cgroup reached its memory limit, if it exists.
+// If the file does not exist or there is no limit, then this will default to 0
+func (c ContainerCgroup) FailedMemoryCount() (unit64, error){
+	v, err := c.ParseSingleStat("memory", "memory.failcnt")
+	if os.IsNotExist(err) {
```

Comment:
Hey @CharlyF, a lot of the functions here submit 0 if does not exist like softmemlimit() and CPUNrThrottled

I think if we accept this as practice, i'll follow along for FailedMemoryCount, but I think we should update documentation to reflect 0 if null like docker.mem.soft_limit in https://docs.datadoghq.com/integrations/docker_daemon/#pagetitle